project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementProteomicsValidator.java

```diff
@@ -95,7 +95,7 @@ public ValidationResult validate(ProteomicsMeasurementMetadata measurementMetada
       return mandatoryValidationResult;
     }
     //If all fields were filled then we can validate the entries individually
-    return validationPolicy.validateSampleIds(measurementMetadata.sampleCodes())
+    return validationPolicy.validateSampleId(measurementMetadata.sampleCode())
```

Comment:
I don't understand this change. As far as I understand it, this method ('validate') ist called by the ValidationService to validate one sample. Previously this would have validated all of them? 
Can you help me understand why this was validating multiple IDs and why this changed?

Reply:
I just noticed the switch from multiple sample ids in the metadata to only one. Please consider this comment solved. This makes the code more clear.